server/src/main/java/org/elasticsearch/action/synonyms/PutSynonymsAction.java

```diff
@@ -118,55 +112,4 @@ public int hashCode() {
             return Objects.hash(synonymsSetId, Arrays.hashCode(synonymRules));
         }
     }
-
-    public static class Response extends ActionResponse implements StatusToXContentObject {
```

Comment:
Refactored this response into a common `SynonymUpdateResponse`, that will be used from both PUT actions

Reply:
You can merge auto-reloading first and I can take care of that later - either reversing the change or making the refactoring after auto-reloading merged